packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -694,6 +696,20 @@ export class RestApi extends RestApiBase {
 
     return [];
   }
+
+  /**
+   * Adds a statement to the resource policy document of the REST API.
+   * The statement must have a Principal.
+   *
+   * @param statement the IAM statement to add
+   */
+  public addToPolicy(statement: iam.PolicyStatement): void {
```

Comment:
Make the parameter a list instead, using the ellipses operator, similar to [`PolicyDocument.addStatements()`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-iam.PolicyDocument.html#add-wbr-statementsstatement)